src/patternfly/components/Nav/nav.scss

```diff
@@ -218,49 +147,24 @@
     position: relative;
     display: flex;
     align-items: baseline;
-    padding:
-      var(--pf-c-nav__list-link--PaddingTop)
-      var(--pf-c-nav__list-link--PaddingRight)
-      var(--pf-c-nav__list-link--PaddingBottom)
-      var(--pf-c-nav__list-link--PaddingLeft);
+    padding: var(--pf-c-nav__list-link--PaddingTop) var(--pf-c-nav__list-link--PaddingX) var(--pf-c-nav__list-link--PaddingBottom);
     font-weight: var(--pf-c-nav__list-link--FontWeight);
     color: var(--pf-c-nav__list-link--Color);
-
-    @media screen and (min-width: $pf-global--breakpoint--md) {
-      --pf-c-nav__list-link--PaddingRight: var(--pf-c-nav__list-link--md--PaddingRight);
-      --pf-c-nav__list-link--PaddingLeft: var(--pf-c-nav__list-link--md--PaddingLeft);
-    }
+    background-color: var(--pf-c-nav__list-link--BackgroundColor);
 
     // States
     &.pf-m-hover,
-    &:hover {
-      color: var(--pf-c-nav__list-link--hover--Color);
-      background-color: var(--pf-c-nav__list-link--hover--BackgroundColor);
-
-      &::after {
-        background-color: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
-      }
-    }
-
+    &:hover,
     &.pf-m-active,
-    &:active {
-      font-weight: var(--pf-c-nav__list-link--active--FontWeight);
-      color: var(--pf-c-nav__list-link--active--Color);
-      background-color: var(--pf-c-nav__list-link--active--BackgroundColor);
-
-      &::after {
-        background-color: var(--pf-c-nav__list-link--active--after--BackgroundColor);
-      }
-    }
-
+    &:active,
     &.pf-m-focus,
     &:focus {
-      font-weight: var(--pf-c-nav__list-link--focus--FontWeight);
-      color: var(--pf-c-nav__list-link--focus--Color);
-      background-color: var(--pf-c-nav__list-link--focus--BackgroundColor);
+      --pf-c-nav__list-link--Color: var(--pf-c-nav__list-link--hover--Color);
+
+      background-color: var(--pf-c-nav__list-link--hover--BackgroundColor);
```

Comment:
can you redefine `--pf-c-nav__list-link--BackgroundColor` here?

Reply:
wasnt needed so removed